devel/gettext/Portfile

```diff
@@ -26,6 +26,10 @@ if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"}
     clang_dependency.extra_versions 3.7
 }
 
+# The patch is applied to whole sources
+patch.dir               ${worksrcpath}
```

Comment:
```suggestion
```

Reply:
@ryandesign it won't work because all subports redefines `worksrcdir` and patches won't applied.

Another approach is create a lot of patches per subport but it is very tricky